kratos/solving_strategies/strategies/implicit_solving_strategy.h

```diff
@@ -0,0 +1,339 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+//
+
+#if !defined(KRATOS_IMPLICIt_SOLVING_STRATEGY)
+#define KRATOS_IMPLICIt_SOLVING_STRATEGY
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "solving_strategy.h"
+#include "solving_strategies/schemes/scheme.h"
+#include "solving_strategies/builder_and_solvers/builder_and_solver.h"
+#include "includes/kratos_parameters.h"
```

Comment:
please only include what is actually used

Reply:
Including the base `solving_strategy.h` should be sufficient. Done.